.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 16]
+        node: [12, 14, 16]
```

Comment:
Should we add node 18 as well since it's current?

Reply:
It's not yet stable, but we will do this once we migrate to 14!